configs/announcement_schema.json

```diff
@@ -17,6 +20,7 @@
     },
     "allocatedPrefix": {
       "$ref": "#/definitions/prefix",
+      "$comment": "Change the 'enum' list to the prefixes allocated for your experiment",
```

Comment:
I forget why we decided to add this; right now I think we can just drop this comment as experimenters are unlikely to edit this file.